stories/fn-list/fn-list.stories.js

```diff
@@ -0,0 +1,168 @@
+export default {
+    title: 'Experimental/List',
+    parameters: {
+        components: ['fn-list', 'icon', 'fn-button', 'fn-checkbox', 'fn-object-status', 'fn-avatar']
+    }
+};
+
+export const Primary = () => `
+    <ul class="fn-list">
+        <li class="fn-list__item" tabindex="0">
+            <span>List Item</span>
```

Comment:
The text will need a class:
```
<span class="fn-list__text">List Item</span>
```
because you need to reset the element, set the line-height, font-size and color.

Reply:
and most importantly, it will allow you to handle truncation